front/.eslintrc

```diff
@@ -106,7 +106,13 @@
     "no-named-as-default": "off",
     "no-param-reassign": "off",
     "no-use-before-define": "off",
-    "no-restricted-syntax": "off",
+    "no-restricted-syntax": [
```

Comment:
Would this typescript-eslint rule be more fitting?

https://typescript-eslint.io/rules/consistent-type-definitions